### DISCUSSION TITLE
Problems with vector receiving data

### DISCUSSION THREAD

**Discussion by taikula-8** [vectordotdev/vector#21017] [Q&A]
![image](https://github.com/user-attachments/assets/fa045959-3d86-4445-84ff-7679badfe5da)

In the vector configuration file set up two sink (are set up the domain name), one of the domain name can not be accessed, the other can be accessed normally, in the vector after the start of the reception of data for a period of time after the data can not be received in the data, but the vector process is still alive in the state, and the source of the port is also listening to the state, how to ask the How to deal with this?

**Comment by jszwedko** [comment]
This sounds like a networking / DNS issue rather than something specific to Vector 🤔
